src/mmrelay/config.py

```
@@ -729,6 +729,35 @@ def is_e2ee_enabled(config: dict[str, Any] | None) -> bool:
     return encryption_enabled or e2ee_enabled
 
 
+def load_config_silently(args: Any = None) -> dict[str, Any]:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `args`

(ANN401)